fhir-term/src/main/java/com/ibm/fhir/term/spi/FHIRTermServiceProvider.java

```diff
@@ -78,4 +95,16 @@
      *     true if the given coding is a member of the specified value set, false otherwise
      */
     boolean validateCode(ValueSet valueSet, Coding coding);
+
+    /**
+     * Indicates whether the given codeable concept contains a coding that is a member of the specified value set
+     *
+     * @param valueSet
+     *     the value set
```

Comment:
any external contract about whether the valueset must be pre-expanded?
assuming not, is it worth mentioning that the service should expand it if needed?

Reply:
Added an `@apiNote` comment to the Javadoc